game_resources.h

```diff
@@ -123,6 +137,18 @@ class game_resources
   /// 'bump' sound from Sebastian
   sf::Music m_bump;
 #endif // IS_ON_TRAVIS
+
+#ifndef IS_ON_TRAVIS
+  // Playing sound on Travis gives thousands of error lines, which causes the
+  // build to fail
+  sf::Music m_grow;
+#endif // IS_ON_TRAVIS
+
```

Comment:
Line 145 and 147 could be removed?

Reply:
I think it is good to keep `m_grow `and `m_shrink `into 2 separate pre-processor statements blocks